### PR TITLE
Acheron Athena support

### PR DESCRIPTION
**Pull request by Gondolindrim**
## Description

Adds support for the Acheron Athena PCBs.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/acheron/athena/alpha/config.h [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+#define BACKLIGHT_LEVELS 20
+#define BACKLIGHT_BREATHING
+#define BREATHING_PERIOD 5
+#define BACKLIGHT_ON_STATE 0
+
+#define STM32_HSECLK 8000000
+
+#define RGB_DI_PIN B15
+#define RGBLED_NUM 34
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/acheron/athena/alpha/config.h:55]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/acheron/athena/alpha/rules.mk [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#
+BOOTMAGIC_ENABLE = yes     # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/acheron/athena/alpha/rules.mk:18]:
```suggestion
COMMAND_ENABLE = yes        # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
```

File: keyboards/acheron/athena/alpha/rules.mk [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
+BACKLIGHT_DRIVER = pwm
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+LTO_ENABLE = yes
+ENCODER_ENABLE = no
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRU
```
Comment (drashna) [inline keyboards/acheron/athena/alpha/rules.mk:26]: I think you missed the E? 
```suggestion
OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```

File: keyboards/acheron/athena/beta/config.h [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+#define BACKLIGHT_PAL_MODE      2
+#define BACKLIGHT_LEVELS 20
+#define BACKLIGHT_BREATHING
+#define BREATHING_PERIOD 5
+
+#define STM32_HSECLK 8000000
+
+#define RGB_DI_PIN B15
+#define RGBLED_NUM 34
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/acheron/athena/beta/config.h:54]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/acheron/athena/beta/rules.mk [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
+BACKLIGHT_DRIVER = pwm
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+LTO_ENABLE = yes
+ENCODER_ENABLE = no
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRU
```
Comment (drashna) [inline keyboards/acheron/athena/beta/rules.mk:26]:
```suggestion
OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```

File: keyboards/acheron/athena/beta/rules.mk [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+#
+BOOTMAGIC_ENABLE = yes     # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/acheron/athena/beta/rules.mk:18]:
```suggestion
NKRO_ENABLE = yes           # Enable N-Key Rollover
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/acheron/athena/alpha/rules.mk (hunk 3 above)
Comment (Gondolindrim) [reply keyboards/acheron/athena/alpha/rules.mk:26]: I missed the E.

File: keyboards/acheron/athena/beta/rules.mk (hunk 6 above)
Comment (Gondolindrim) [reply keyboards/acheron/athena/beta/rules.mk:18]: Remove comments from rules.mk

File: keyboards/acheron/athena/readme.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
+# Acheron Athena QMK firmware
+
+![athena](https://acheronproject.com/images/athena/IMG-9653.jpg)
```
Comment (fauxpark) [inline keyboards/acheron/athena/readme.md:3]: Dead link.

File: keyboards/acheron/athena/beta/rules.mk [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
+BACKLIGHT_DRIVER = pwm
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+LTO_ENABLE = yes
+ENCODER_ENABLE = no
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```
Comment (fauxpark) [inline keyboards/acheron/athena/beta/rules.mk:23]:
```suggestion

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
BACKLIGHT_DRIVER = pwm
LTO_ENABLE = yes
ENCODER_ENABLE = no

# Enter lower-power sleep mode when on the ChibiOS idle thread
OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```

File: keyboards/acheron/athena/alpha/rules.mk [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
+BACKLIGHT_DRIVER = pwm
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+LTO_ENABLE = yes
+ENCODER_ENABLE = no
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```
Comment (fauxpark) [inline keyboards/acheron/athena/alpha/rules.mk:23]:
```suggestion

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
BACKLIGHT_DRIVER = pwm
LTO_ENABLE = yes
ENCODER_ENABLE = no

# Enter lower-power sleep mode when on the ChibiOS idle thread
OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/acheron/athena/readme.md (hunk 7 above)
Comment (Gondolindrim) [reply keyboards/acheron/athena/readme.md]: Fixed in cfae803

File: keyboards/acheron/athena/beta/rules.mk (hunk 8 above)
Comment (Gondolindrim) [reply keyboards/acheron/athena/beta/rules.mk]: Applied in 8be3b70

File: keyboards/acheron/athena/alpha/rules.mk (hunk 9 above)
Comment (Gondolindrim) [reply keyboards/acheron/athena/alpha/rules.mk]: Applied in 8be3b70

File: keyboards/acheron/athena/readme.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
+# Acheron Athena QMK firmware
+
+![athena](https://acheronproject.com/images/athena/IMG_9653.jpg)
```
Comment (fauxpark) [inline keyboards/acheron/athena/readme.md:3]: Please use a smaller image here.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/acheron/athena/alpha/chconf.h [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
```
Comment (drashna) [inline keyboards/acheron/athena/alpha/chconf.h:17]: Is this file actually needed?

File: keyboards/acheron/athena/beta/chconf.h [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
```
Comment (drashna) [inline keyboards/acheron/athena/beta/chconf.h:17]: Is this file needed?

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/acheron/athena/readme.md (hunk 10 above)
Comment (Gondolindrim) [reply keyboards/acheron/athena/readme.md:3]: The link has been updated with a reduced image.

File: keyboards/acheron/athena/alpha/chconf.h (hunk 11 above)
Comment (Gondolindrim) [reply keyboards/acheron/athena/alpha/chconf.h]: Removed in 257aff4

File: keyboards/acheron/athena/beta/chconf.h (hunk 12 above)
Comment (Gondolindrim) [reply keyboards/acheron/athena/beta/chconf.h]: Removed in 7033c0c

File: keyboards/acheron/athena/readme.md (hunk 10 above)
Comment (fauxpark) [reply keyboards/acheron/athena/readme.md:3]: Doesn't seem to have been changed.
image

Comment (Gondolindrim) [reply keyboards/acheron/athena/readme.md:3]: It did, not in pixel size but density. Size now is ten times less.

**Review by drashna — APPROVED** [review]

File: keyboards/acheron/athena/readme.md (hunk 10 above)
Comment (fauxpark) [reply keyboards/acheron/athena/readme.md:3]: Not really what I meant by smaller...

Comment (Gondolindrim) [reply keyboards/acheron/athena/readme.md:3]: Sorry. I giggled

Reduced it to 20% pixel size

Comment (Gondolindrim) [reply keyboards/acheron/athena/readme.md:3]: Ok I reduced it to 20% pixel size lol

**Review by fauxpark — APPROVED** [review]
